src/op_mode/generate_firewall_rule-resequence.py

```diff
@@ -119,6 +119,11 @@ def convert_rule_keys_to_int(config_dict):
     #config_dict =  config.get_config_dict('firewall')
     config_dict = config.get_config_dict('firewall')
 
+    # Remove global-options, flowtable and group as they don't need sequencing
+    del config_dict['firewall']['global-options']
```

Comment:
You should check if those options exist in the dictionary.
For example, I don't have `global-options`
```
vyos@r4:~$ generate firewall rule-resequence 
Traceback (most recent call last):
  File "/usr/libexec/vyos/op_mode/generate_firewall_rule-resequence.py", line 123, in <module>
    del config_dict['firewall']['global-options']
        ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
KeyError: 'global-options'
vyos@r4:~$ 

```

Reply:
Thanks for catching by newbie mistake :) 

Added proposed change. I don't have flowtable in my config and it doesn't throw an error now. I forgot to test that one after I added it last time.